tools/Customization/DevHome.Customization/Views/DevDriveInsightsView.xaml

```diff
@@ -15,39 +17,55 @@
 
-            <converters:BoolNegationConverter x:Key="BoolNegationConverter" />
             <converters:BoolToVisibilityConverter x:Key="CollapsedWhenTrueBoolToVisibilityConverter" TrueValue="Collapsed" FalseValue="Visible"/>
-            <converters:EmptyCollectionToObjectConverter x:Key="NegatedEmptyCollectionVisibilityConverter" EmptyValue="Visible" NotEmptyValue="Collapsed"/>
-            <converters:EmptyCollectionToObjectConverter x:Key="EmptyCollectionVisibilityConverter" EmptyValue="Collapsed" NotEmptyValue="Visible"/>
-            <converters:EmptyObjectToObjectConverter x:Key="EmptyObjectToObjectConverter" NotEmptyValue="Visible" EmptyValue="Collapsed"/>
-            <converters:EmptyStringToObjectConverter x:Key="EmptyStringToObjectConverter" NotEmptyValue="Visible" EmptyValue="Collapsed"/>
 
             <!--- Template for dev drive horizontal cards on the dev insights page. -->
             <DataTemplate x:Key="HorizontalCardForDevDriveInsightsPage" x:DataType="devViewModels:DevDriveCardViewModel">
-                <ctControls:SettingsCard Header="{Binding DevDriveLabel, Mode=OneWay}">
-                    <ctControls:SettingsCard.HeaderIcon>
+                <ctControls:SettingsExpander IsExpanded="False" Header="{Binding DevDriveLabel, Mode=OneWay}" >
+                    <ctControls:SettingsExpander.Description>
+                        <TextBlock Text="{Binding Path=DevDriveTrustText, Mode=OneWay}" Foreground="{ThemeResource SystemFillColorCautionBrush}">
+                            <i:Interaction.Behaviors>
+                                <ic:DataTriggerBehavior Binding="{x:Bind IsDevDriveTrusted}" Value="True">
+                                    <ic:ChangePropertyAction PropertyName="Foreground" Value="{ThemeResource SystemFillColorSuccessBrush}"/>
+                                </ic:DataTriggerBehavior>
+                            </i:Interaction.Behaviors>
+                        </TextBlock>
+                    </ctControls:SettingsExpander.Description>   
+                    <ctControls:SettingsExpander.HeaderIcon>
                         <FontIcon Glyph="&#xEDA2;" FontSize="25" FontFamily="{ThemeResource AmcFluentIcons}"/>
-                    </ctControls:SettingsCard.HeaderIcon>
+                    </ctControls:SettingsExpander.HeaderIcon>
                     <Grid Grid.Column="2">
                         <Grid.RowDefinitions>
                             <RowDefinition Height="Auto"  />
                             <RowDefinition Height="Auto" />
                             <RowDefinition Height="Auto" />
                         </Grid.RowDefinitions>
                         <TextBlock Style="{StaticResource CardBodyStackPanelAltTextBlockStyle}" HorizontalAlignment="Right" Margin="0 0 115 5">
-                                <Run Text="{Binding Path=DevDriveSizeText, Mode=OneWay}" />
+                            <Run Text="{Binding Path=DevDriveSizeText, Mode=OneWay}" />

```

Comment:
Can `Size` be bound to the TextBlock instead?

Reply:
Yes, this has been fixed in my other PR.